.github/dependabot.yml

```diff
@@ -11,3 +11,11 @@ updates:
           - "*"  # Group all Actions updates into a single larger pull request
     schedule:
       interval: weekly
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: docker
+    directory: docker
+    schedule:
+      interval: weekly
```

Comment:
I appreciate your work on this, but we've intentionally turned off dependabot in iroh in the past as it didn't fit into our workflow.